components/infobox/commons/infobox_basic.lua

```diff
@@ -31,4 +34,49 @@ function BasicInfobox:createBottomContent(infobox)
     return nil
 end
 
+--- Allows for using this for customCells
+function BasicInfobox:getMultiArgsForType(args, argType)
```

Comment:
These two functions are identical, should just use an options table as param and combine the two